src/esp/sensor/CameraSensor.cpp

```diff
@@ -38,6 +44,15 @@ void CameraSensor::setProjectionParameters(const SensorSpec::ptr& spec) {
 
 }  // setProjectionParameters
 
+void CameraSensor::recomputeProjectionMatrix() {
+  projectionMatrix_ = zoomMatrix_ * baseProjMatrix_;
+  // update renderCamera_'s projectionMatrix every time the sensor's
+  // projectionMatrix changes
+  if (renderCamera_) {
```

Comment:
Is there any scenario where the CameraSensor instance *doesn't* have a renderCamera?